### PR TITLE
Rename codimd to hedgedoc

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
Almost everything

### Description
This PR replaces every occurrence of "codimd" with "hedgedoc"

### Steps

- [x] added implementation
- [x] added / updated tests
- [x] added / updated documentation
- [x] extended changelog

### CODE REVIEW

**Comment by DerMolly** [comment]
Is something missing from this PR? If not we should probably merge it and talk about HedgeDoc everywhere in the new client

**Comment by ErikMichelson** [comment]
The domains {community,translate}.hedgedoc.org are not existing yet. But that shouldn't be problematic and merge-blocking.

**Comment by mrdrogdrog** [comment]
This PR is using a draft image as logo. I still don't know when we will continue with the logo decision, so I don't know if we should wait to merge everything or merge the texts now.

@codimd/developers

**Comment by DerMolly** [comment]
merging texts now would be good, as that includes the i18n files, which we want to import into POEditor soon. And it would be nice to have all instances of CodiMD replaced by HedgeDoc before that…

**Review by DerMolly — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,9 +30,9 @@ HedgeDoc is a volunteer effort. We encourage you to pitch in and to help us maki
 
 Thanks! :heart: :heart: :heart:
 
-[issues]: https://github.com/codimd/react-client/issues
-[new_issue]: https://github.com/codimd/react-client/issues/new/choose
+[issues]: https://github.com/hedgedoc/react-client/issues
+[new_issue]: https://github.com/hedgedoc/react-client/issues/new/choose
```
Comment (ErikMichelson) [inline CONTRIBUTING.md:34]: Do we want to change this already? The new repo URL doesn't exist yet and as it's our contributing information, it should probably be correct.

**Review by ErikMichelson — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (mrdrogdrog) [reply CONTRIBUTING.md]: Good point. I changed that back.

**Review by ErikMichelson — APPROVED** [review]

**Review by ErikMichelson — APPROVED** [review]

**Pull request merged**